package.json

```diff
@@ -10,25 +8,39 @@
     "scss",
     "web component"
   ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/BrightspaceUI/colors.git"
+  },
+  "name": "d2l-colors",
   "scripts": {
-    "postinstall": "polymer install --variants",
     "build:sass": "node-sass ./test/test.scss ./test/test.css",
-    "test:lint": "npm run test:lint:wc && npm run test:lint:html",
-    "test:lint:html": "eslint *.html",
+    "test:lint": "npm run test:lint:wc && npm run test:lint:js",
+    "test:lint:js": "eslint . test/** demo/** --ext .js,.html",
     "test:lint:wc": "polymer lint",
     "test": "npm run test:lint && npm run build:sass"
   },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/BrightspaceUI/colors.git"
-  },
   "author": "D2L Corporation",
   "license": "Apache-2.0",
   "devDependencies": {
+    "@babel/polyfill": "^7.0.0",
+    "@webcomponents/webcomponentsjs": "^2.1.3",
+    "babel-eslint": "^10.0.1",
     "eslint": "^4.15.0",
     "eslint-config-brightspace": "^0.4.0",
     "eslint-plugin-html": "^4.0.1",
```

Comment:
I _think_ we should be able to remove `eslint-plugin-html` plugin.  The original purpose of this plugin was to extract  JS from `.html` files during linting - specifically, web component JS.  This was because core eslint could not be pointed at `.html` files.  The only `.html` file that remains is the `demo/index.html`.

Will probably need to remove [the plugin](https://github.com/Brightspace/eslint-config-brightspace/blob/master/polymer-3-config.js#L8) from the P3 eslint config.

Reply:
I think the plugin would still be useful in the P3 eslint config because the test files still use html. Although they can be moved to JS.

Reply:
Agree with @awikkerink -- the demo files also would get linted with the plugin, which is nice.